ext/pcntl/tests/pcntl_skipif_user_env_rules.inc

```diff
@@ -0,0 +1,11 @@
+<?php
+
+$envUser = getenv("USER") ?: getenv("USERNAME");
+
+if ($envUser === false) {
+    die("skip This test is not executed without environment variables USER/USERNAME");
+} elseif ($envUser === "root") {
+    die("skip This test is not executed with root user");
+}
```

Comment:
Hmm, is that really suitable? I'm especially referring to checking for "root", since to my knowledge that username can be changed. Wouldn't it be better to check for `posix_geteuid() === 0` (although that would require the posix extension for some pcntl tests).

Reply:
Good point. I Will add such a check also